bb/integration_tests.clj

```diff
@@ -49,6 +49,12 @@
   (is (str/starts-with? (obb* "-e" (pr-str '(js/Application "Safari")))
                         "#org.babashka.obb/object-specifier")))
 
+(deftest version-test
+  (is (re-matches #"obb v[\d]+\.[\d]+\.[\d]+(\-SNAPSHOT)?\n"
+                  (-> (obb** "--version")
+                      check
+                      :err))))
```

Comment:
I would say --version should print to `stdout` as that is info that tools might want to parse.

Reply:
Changed!